staging/src/k8s.io/apimachinery/pkg/runtime/serializer/codec_factory.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	_ "k8s.io/apimachinery/pkg/runtime/serializer/cbor"
```

Comment:
The impact isn't as substantial having now switched libraries, but I've moved the import into a test package so that kube component binary sizes won't change at all for now.